DESCRIPTION

```diff
@@ -7,12 +7,18 @@ URL: https://bioconductor.org/packages/Biobase
 BugReports: https://github.com/Bioconductor/Biobase/issues
 Version: 2.57.1
 License: Artistic-2.0
-Author: R. Gentleman, V. Carey, M. Morgan, S. Falcon
+Authors@R: c(
+    person("R.", "Gentleman", role="aut"),
+    person("V.", "Carey", role = "aut"),
+    person("M.", "Morgan", role="aut"),
+    person("S.", "Falcon", role="aut"),
+    person("Haleema", "Khan", role = "ctb"))
 Maintainer: Bioconductor Package Maintainer
```

Comment:
The `Authors@R` format should include the maintainer (`role = "cre"`). I think the following works and should be added to the `Authors@R` entry

```
    person(
        "Bioconductor Package Maintainer",
        role = "cre",
        email = "maintainer@bioconductor.org"
    )
```
The line `Maintainer: Bioconductor Package Maintainer` and email on the following line should be removed.

Reply:
Hello, I see that it is merged now. What should I do. making the changes as per the latest review, will it affect anything?

Reply:
@hsadia538  Could I ask you to make another pull request with these changes? @mtmorgan Thanks for your input, which helps me a lot, and your advocacy to make sure contributors get credit.

Reply:
Sure thanks for clarifying. I will make a new one.